cli/dcoscli/package/main.py

```diff
@@ -180,17 +180,25 @@ def _add_repo(repo_name, repo_url, index):
     return 0
 
 
-def _remove_repo(repo_name):
+def _remove_repo(repo_names):
     """Remove package repo and update repo with new repo
 
-    :param repo_name: name to call repo
-    :type repo_name: str
+    :param repo_names: names of repos
+    :type repo_name: str or [str]
     :returns: Process status
     :rtype: int
     """
 
+    try:
+        # Try treating repo_name as a string
+        repo_names = repo_names.split(',')
+    except AttributeError:
```

Comment:
Oh, the reason for this was that there were some tests that were failing because they were calling this function directly and passing it a single string.

We could just fix those tests and then we wouldn't need this ugliness here.